app/config/security.yml

```diff
@@ -2,10 +2,28 @@
 # https://symfony.com/doc/current/security.html
 security:
 
+    encoders:
+        p6\UserBundle\Entity\User:
+            algorithm: bcrypt
+
+
+    role_hierarchy:
+        ROLE_ADMIN:       ROLE_USER
+
     # https://symfony.com/doc/current/security.html#b-configuring-how-users-are-loaded
     providers:
-        in_memory:
-            memory: ~
+        #in_memory:
```

Comment:
Can be deleted. No need to keep it commented

Reply:
Code delete